agent/src/main/java/com/appland/appmap/record/Recorder.java

```diff
@@ -161,6 +163,8 @@ public void add(Event event) {
       return;
     }
 
+    Logger.println("Recorder.add, event: " + JSON.toJSONString(event));
```

Comment:
Would it make sense to optimize out this JSON serialization if the log level is not enabled?

Reply:
Oops, didn't mean to leave that in there at all. :)